docs/getting-started/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Install Shopify App CLI
+section: getting-started
+toc: false
+redirect_from: "/install/"
+---
+
+Shopify App CLI can be installed using a variety of package managers.
+
+### Brew (MacOS)
+
+You’ll need to run `brew tap` first to add Shopify’s third-party repositories to Homebrew.
+
+```console
+$ brew tap shopify/shopify
+$ brew install shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+You’ll need to install a downloaded .deb file with an explicit version number. Make sure you check the [releases page](https://github.com/Shopify/shopify-app-cli/releases) to make sure you install the latest package.
+
+```console
+$ sudo apt install shopify-cli-x.y.z.deb
+```
+
+### RPM (CentOS, Fedora, Red Hat, SUSE)
```

Comment:
And then this heading should be "RPM-based Linux distributions (e.g., Fedora, CentOS, SUSE)"